modules/gsp-namespace-policies/iam.tf

```
@@ -8,7 +8,7 @@ data "aws_iam_policy_document" "namespace-sqs" {
       "sqs:DeleteMessage"
     ]
 
-    resources = ["arn:aws:sqs:*:${var.account_id}:${var.cluster_name}-sqsqueue-${var.namespace_name}-*"]
+    resources = ["arn:aws:sqs:*:${var.account_id}:${var.cluster_name}-sqsqueue-*"]
```

I suspect we might need:
```suggestion
    resources = ["arn:aws:sqs:*:${var.account_id}:*"]
```
but let's go with what you currently have and see if it works